### PR TITLE
FIX: 'passwordsent' title was not being displayed (fixes #10366)

### LINKED ISSUE #10366: "Password reset link sent" title is never shown
## Affected Version

4.7+

## Description

After completing the password reset form and being redirected to `/Security/lostpassword/passwordsent`, the same title of “Lost password” shows instead of “Password reset link sent”.

I think this is just a case of swapping the arguments over here:

https://github.com/silverstripe/silverstripe-framework/blob/139fa07bcb5f3b4a4ca2c5ef00c957ff588b07ba/src/Security/Security.php#L951

PR: https://github.com/silverstripe/silverstripe-framework/pull/10367

### PR DESCRIPTION
**Pull request by kinglozzer**
Fixes #10366

### CODE REVIEW

File: src/Security/Security.php [hunk 1]
```diff
@@ -948,7 +948,7 @@ protected function renderWrappedController($title, array $fragments, array $temp
         static::clearSessionMessage();
 
         // Ensure title is present - in case getResponseController() didn't return a page controller
-        $fragments = array_merge($fragments, ['Title' => $title]);
+        $fragments = array_merge(['Title' => $title], $fragments);
```
Comment (GuySartorelli) [inline src/Security/Security.php:951]:
```suggestion
        $fragments['Title'] = $title;
```
Probably makes more sense to just set that in the array directly.

Comment (Cheddam) [reply src/Security/Security.php:951]: The merge allows an existing title key in the fragment array to take precedence, so it needs to stay put.

**Review by Cheddam — APPROVED** [review]
I think the commit message could do with a bit more explanation of the change, and a test case is always welcome, but otherwise this LGTM.

File: src/Security/Security.php (hunk 1 above)
Comment (GuySartorelli) [reply src/Security/Security.php:951]: Ahh, for some reason I had that backwards. Fair enough.

**Pull request merged**